### ISSUE TITLE
tensorflow Load model gives the random prediction in python new session.(LSTM.)

### ISSUE THREAD

**Issue by samrajkodai** [tensorflow/tensorflow#57661]
after train and test the model gives the good accuracy and perfect predictions.

model.save()
load_model()
once i save the model and testing in a new python session it gives random prediction.

i tried many methods but I'm not able to sole the issue. anyone have a solution?

**Comment by samrajkodai** [comment]
after train and test the model gives the good accuracy and perfect predictions.

once i save the model and testing in a new python session it gives random prediction.

i tried many methods but I'm not able to sole the issue. anyone have a solution?

**Comment by tilakrayal** [comment]
@samrajkodai,
To expedite the trouble-shooting process, could you please provide a complete code and the TensorFlow version you are using. Thank you!

**Comment by samrajkodai** [comment]
hi,

hereby i have attached the code, actually after training and saving this model works fine but when i open new python terminal and load the model for testing it gives  0% accuracy.

my tensorflow version is  2.11.0-dev20220812

[issue.txt](https://github.com/tensorflow/tensorflow/files/9597363/issue.txt)

**Comment by samrajkodai** [comment]
i checked that also label id's are same only.

**Comment by samrajkodai** [comment]
https://github.com/keras-team/keras/issues/4875 

i have tried this also but my problem is not solved.

**Comment by tilakrayal** [comment]
@samrajkodai,
Thanks for opening this issue. Development of keras moved to another [repository](https://github.com/keras-team/keras/issues). 
 
 
Could you please post this issue on keras-team/keras [repo](https://github.com/keras-team/keras/issues).
To know more please refer:
https://discuss.tensorflow.org/t/keras-project-moved-to-new-repository-in-https-github-com-keras-team-keras/1999
Thank you!

**Comment by samrajkodai** [comment]
@tilakrayal 

Thankyou for yor response.

**Comment by tilakrayal** [comment]
@samrajkodai,
Could you please feel free to move this issue to closed status, since it is already being tracked in Keras repo. Thank you!

**Comment by HebaGamalElDin** [comment]
Hi @samrajkodai , I have the same issue.. would you please till me if you fixed this?! thanks in advance!

**Comment by samrajkodai** [comment]
hi @HebaGamalElDin i did'nt i tried many methods but i am not able to solve this issue still facing problem

**Comment by samrajkodai** [comment]
@HebaGamalElDin hi please check this, https://github.com/keras-team/keras/issues/4875#issuecomment-1326294523  it may be solve your problem.
